client/src/components/Layout/Layout.js

```diff
@@ -1,16 +1,48 @@
-import React from "react";
+import React, { Component } from "react";
 import "./Layout.css";
 import Navbar from "../Navigation/Navbar/Navbar";
+import { _isLoggedIn } from "../../services/NavbarService";
 
-const Layout = props => {
+class Layout extends Component {
+  constructor() {
+    super()
+    this.state = {
+      user: false
+    }
+
+  }
+
+  componentDidMount (){
+    
+    return _isLoggedIn (localStorage.getItem('token')).then(user => {
+      debugger;
+      if(user.message == "Right Token"){
+  
+        this.setState({
+          user: true
+        })
+      }else {
+        this.setState({
+          user: false
+        })
+      }
+    })
+ 
+
+  }
+
+
+  render (){
   return (
     <div>
-      <Navbar />
+      {this.state.user && <Navbar /> }
```

Comment:
@avakrishn Are we hiding the whole nav bar?

Reply:
Yes when not logged in. It will make sense when it is merged. I am not ready to merge yet. I needed to create a PR to resolve conflicts. 
I see that you have been merging your own PRs.

Reply:
yes the pr I merged was the resolve of broken styling issue on the navbar when I merged Hooman's branch and pull Jason's commit at the same time. It's a hot fix. I found it no use to have you or Yanan look at it since I reviewed Hooman and Jason's commits

Reply:
I had to fix a lot of things last night on the master branch